libdevcore/Exceptions.h

```diff
@@ -42,7 +42,7 @@ struct Exception: virtual std::exception, virtual boost::exception
 private:
 };
 
-#define DEV_SIMPLE_EXCEPTION(X) struct X: virtual Exception { const char* what() const noexcept override { return #X; } }
+#define DEV_SIMPLE_EXCEPTION(X) struct X: Exception { const char* what() const noexcept override { return #X; } }
```

Comment:
Note that this here is virtual inheritance, not a virtual function override, so this ``virtual`` is *not* redundant and we should keep it.

This will make a difference, if someone defines a new exception type that inherits from more than one of the exceptions defined using this macro - I don't think this is done at the moment, but nevertheless it makes sense to use virtual inheritance here.

Reply:
Exactly. virtual inheritance is used to eliminate base classes in multiple inheritance scenarios (rarely used, but here it must not be removed. Please undo this change.) :-)

Reply:
Fixed.

Reply:
Yeah, sorry about that. Thanks for fixing it @axic.